data/elasticsearch.mappings

```diff
@@ -0,0 +1,77 @@
+{
```

Comment:
Where does this come from? Do we generate this mapping? It's just a subset of the attributes avaialble so I don't see the need for this. ES should map correctly by default in these cases.

Reply:
this comes from the elastic output module https://github.com/log2timeline/plaso/pull/3314/files/f6652638bb3da6c69917cd8142108d907a56677d#diff-48a52c9b262a7ecbed3950483191eeec5492f0983c87eb7e580e5a6656623efeL17 I think @kiddinn implemented this back in the day

Reply:
Correction this comes from https://github.com/log2timeline/plaso/commit/6a5f6e918203bfa126f4d0572bbeb5fed67b208a

Reply:
This change was made to work-around bigint issue seen with some entries.

Reply:
Ah, I see. Ok, makes sense.